app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,32 @@
  *= require_tree .
  *= require_self
  */
+
+@import "components/buttons";
+@import "components/footer";
+@import "components/links";
+@import "components/navbar";
+@import "config/colors";
+@import "config/fonts";
+@import "config/rotations";
+@import "config/spaces";
+@import "pages/homepage.css";
```

Comment:
I have not. Would I check this by checking the server? Could you maybe explain to me what I'm looking for?

I tested with sprocket's require but it seems to take longer, could be me misreading it though:
– with require:
`Completed 200 OK in 24ms (Views: 23.4ms | Allocations: 18234)`
– with import:
`Completed 200 OK in 4ms (Views: 3.6ms | Allocations: 4136)`

Also, not sure if it's related, but what do you think about working with sass, adding the 'sassc-rails' gem?

Reply:
> I have not. Would I check this by checking the server? Could you maybe explain to me what I'm looking for?

The network tab. When using import in pure CSS, it makes an individual request for each `import`.

> Also, not sure if it's related, but what do you think about working with sass, adding the 'sassc-rails' gem?

Not related, but I'd prefer it, since it adds some convenience without having to overblow the project with stuff like webpacker.

Reply:
I looked at the network tab, but it looks like it's not requesting each one individually:

<img width="993" alt="Screen Shot 2022-09-02 at 13 51 24" src="https://user-images.githubusercontent.com/87862340/188202001-9cb0dd89-af4b-4122-804d-4e1247222f44.png">

Reply:
Then there's nothing to worry about :D